docs-source/content/samples/simple/elastic-stack/soa-domain/weblogic-monitoring-exporter-setup.md

```diff
@@ -0,0 +1,305 @@
+
+* `wls-exporter.war`
+* `getX.X.X.sh`
+
+#### Create a configuration file for the WebLogic Monitoring Exporter
+
+The configuration file will have the server port for the web application metrics to be exported from the WebLogic Server.
+
+See the following sample snippet of the configuration:
+
+```
+metricsNameSnakeCase: true
+restPort: 7001
+queries:
+- key: name
+  keyName: location
+  prefix: wls_server_
+  applicationRuntimes:
+    key: name
+    keyName: app
+    componentRuntimes:
+      prefix: wls_webapp_config_
+      type: WebAppComponentRuntime
+      key: name
+      values: [deploymentState, contextRoot, sourceInfo, openSessionsHighCount, openSessionsCurrentCount, sessionsOpenedTotalCount, sessionCookieMaxAgeSecs, sessionInvalidationIntervalSecs, sessionTimeoutSecs, singleThreadedServletPoolSize, sessionIDLength, servletReloadCheckSecs, jSPPageCheckSecs]
+      servlets:
+        prefix: wls_servlet_
+        key: servletName
+
+- JVMRuntime:
+    prefix: wls_jvm_
+    key: name
+
+- executeQueueRuntimes:
+    prefix: wls_socketmuxer_
+    key: name
+    values: [pendingRequestCurrentCount]
+
+- workManagerRuntimes:
+    prefix: wls_workmanager_
+    key: name
+    values: [stuckThreadCount, pendingRequests, completedRequests]
+
+- threadPoolRuntime:
+    prefix: wls_threadpool_
+    key: name
+    values: [executeThreadTotalCount, queueLength, stuckThreadCount, hoggingThreadCount]
+
+- JMSRuntime:
+    key: name
+    keyName: jmsruntime
+    prefix: wls_jmsruntime_
+    JMSServers:
+      prefix: wls_jms_
+      key: name
+      keyName: jmsserver
+      destinations:
+        prefix: wls_jms_dest_
+        key: name
+        keyName: destination
+
+- persistentStoreRuntimes:
+    prefix: wls_persistentstore_
+    key: name
+- JDBCServiceRuntime:
+    JDBCDataSourceRuntimeMBeans:
+      prefix: wls_datasource_
+      key: name
+- JTARuntime:
+    prefix: wls_jta_
+    key: name
+```
+
+#### Generate the deployment package
+
+You must generate two separate deployment packages with the `restPort` as `7001` or `8001` in the `config.yaml` file.
+Two packages are required because the listening ports are different for the Administration Server and Managed Servers.
+
+Use the `getX.X.X.sh` script to update the configuration file into the `wls-exporter` package.  
+See the following sample usage:
+
+```
+$ ./get1.1.0.sh config-admin.yaml
+ % Total % Received % Xferd Average Speed Time Time Time Current
+ Dload Upload Total Spent Left Speed
+100 607 0 607 0 0 915 0 --:--:-- --:--:-- --:--:-- 915
+100 2016k 100 2016k 0 0 839k 0 0:00:02 0:00:02 --:--:-- 1696k
+created /tmp/ci-H1SNbxKo1b
+/tmp/ci-H1SNbxKo1b ~/weblogic-monitor/tmp
+in temp dir
+ adding: config.yml (deflated 63%)
+~/weblogic-monitor/tmp
+$ ls
+config-admin.yaml get1.1.0.sh wls-exporter.war
+```
+
+Similarly, you must generate the deployment package for the Managed Servers and cluster with a different configuration file.
```

Comment:
please remove "and cluster" from line 158.

Reply:
Done.